train_segmentation_model.py

```diff
@@ -9,20 +9,37 @@
 import pytz
 import matplotlib.pyplot as plt
 import ipykernel    # needed when using many metrics, to avoid automatic verbose=2 output
-from tensorflow.keras.callbacks import ModelCheckpoint, TensorBoard, CSVLogger
+from tensorflow.keras.callbacks import ModelCheckpoint, TensorBoard, CSVLogger, Callback
 from image_utils import TensorBoardImage, ImagesAndMasksGenerator
 import git
 from gcp_utils import copy_folder_locally_if_missing
 from models import generate_compiled_segmentation_model
 from metrics_utils import global_threshold
 from local_utils import local_folder_has_files, getSystemInfo, getLibVersions
+import time
+import csv
```

Comment:
is this still needed?

Reply:
time is, csv is not, you are right. removed